public/script.js

```diff
@@ -22,8 +22,19 @@ const DEFAULTS = {
 };
 
 let map;
+const validDates = [];
 /** Main function run to start animation */
 async function main() {
+  await fetch('https://nsidc.org/api/mapservices/NSIDC/wms?service=wms&version=1.1.1&request=GetCapabilities').then(function(response) {
```

Comment:
Add loading message for the black screen before the map loads.

Reply:
Utilize the same method for displaying the loading screen and for the "no data" screen.